pkg/sql/distsql_physical_planner.go

```diff
@@ -1013,8 +1013,8 @@ func tableOrdinal(
 	}
 	if visibility == execinfra.ScanVisibilityPublicAndNotPublic {
 		offset := len(desc.Columns)
-		for i, col := range desc.MutationColumns() {
-			if col.ID == colID {
+		for i := range desc.MutationColumns() {
```

Comment:
extract `desc.MutationColumns()` outside of the loop, I don't think the compiler can necessarily eliminate the common subexpression here.